bower.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "CarLog",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "dependencies": {
-    "angular": "1.x.x",
-    "bootstrap": "3.x.x",
+    "angular": "1.3.14",
```

Comment:
It will download the latest if it can, but it also tries to meet the
requirements for the others, and with the bootstrap-css-only it needs
1.2.x. Ngroute wanted 1.3.x. You can try changing angular back to 1.x.x,
deleting the bower folder, and reinstalling to see what I mean. I don't
remember trying "1.3.x".
On Mar 8, 2015 8:25 PM, "Alex Memering" <notifications@github.com> wrote:

> In bower.json
> <https://github.com/Memeriaj/CarLog/pull/1#discussion_r26011219>:
>
> >    "dependencies": {
> > -    "angular": "1.x.x",
> > -    "bootstrap": "3.x.x",
> > +    "angular": "1.3.14",
>
> Really? You know that it will select the highest version available when
> you give it an x in the minor or patch portions of the version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Memeriaj/CarLog/pull/1/files#r26011219>.
>